stumpy/stumped.py

```diff
@@ -124,13 +131,27 @@ def stumped(dask_client, T_A, m, T_B=None, ignore_trivial=True):
     M_T, Σ_T = core.compute_mean_std(T_A, m)
     μ_Q, σ_Q = core.compute_mean_std(T_B, m)
 
+    T_A[np.isnan(T_A)] = 0
```

Comment:
Remind me, what is the logic here again?

Reply:
See below

Reply:
Perhaps this comment should be added here?

# Remove all nan values from T_A only after first profile is calculated